webapp/components/Map.tsx

```
@@ -1,27 +1,78 @@
 
-import { useEffect, useRef } from "react";
+import { useEffect } from "react";
+
+import ReactMapGl, { MapLayerMouseEvent } from "react-map-gl/maplibre";
 import maplibregl from "maplibre-gl";
 import "maplibre-gl/dist/maplibre-gl.css";
+import { Protocol } from "pmtiles";
+import layers from "protomaps-themes-base";
+import { MAPLIBRE_MAP } from "@/app/config";
 
 export default function Map() {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)